.github/workflows/close-stale-feature-requests.yml

```
@@ -0,0 +1,43 @@
+name: Close stale feature requests
+on:
+  schedule:
+    - cron: 0 1 * * *
+
+env:
+  CLOSE_MESSAGE: >
+    There has been no activity on this feature request
+    and it is being closed. We value your input
+    and if you feel closing this issue was not the
+    right thing to do, please re-open it.
```

People often can't re-open an issue, especially if they are not the one who opened it, so asking people to leave a comment might be better?

```suggestion
    right thing to do, please leave a comment.
```